forestserviceprototype/specialuseform/forms.py

```diff
@@ -2,14 +2,43 @@
 from localflavor.us import forms as localflavor
 from .models import Permit
 import floppyforms.__future__ as forms
+from form_utils.forms import BetterModelForm
 
-class PermitForm(forms.ModelForm):
+class PermitForm(BetterModelForm):
     class Meta:
         model = Permit
-        fields = ['event_name', 'organizer_address_1', 'organizer_address_2', 'city', 'state', 'zipcode', 'phone_daytime', 'phone_evening', 'description', 'location', 'participant_number', 'spectator_number', 'start_date', 'end_date', 'permit_holder_name', 'permit_holder_signature']
+        fieldsets = [
+        ('general', {
+                'fields': ['event_name', 'organizer_address_1', 'organizer_address_2', 'city', 'state', 'zipcode', 'email', 'phone_daytime', 'phone_evening'],
+                'legend': 'Organizer Information',
+            }),
+        ('event_details', {
+                'fields': ['description', 'location', 'participant_number', 'spectator_number', 'start_date', 'end_date'],
+                'legend': 'Event Details',
+            }),
+        ('primary_permit_holder', {
+                'fields': ['permit_holder_name', 'permit_holder_address_1', 'permit_holder_address_2', 'permit_holder_city', 'permit_holder_state', 'permit_holder_zipcode', 'permit_holder_signature'],
+                'legend': 'Primary Permit Holder Information',
+                'classes': ['permit_holder_1_fieldset']
+            }),
+        ('secondary_permit_holder', {
+                'fields': ['permit_holder_2_name', 'permit_holder_2_address_1', 'permit_holder_2_address_2', 'permit_holder_2_city', 'permit_holder_2_state', 'permit_holder_2_zipcode', 'permit_holder_2_signature'],
+                'legend': 'Secondary Permit Holder Information',
+                'classes': ['permit_holder_fieldset', 'permit_hide']
+            }),
+        ]
         widgets = {
-#            'start_date': forms.SelectDateWidget,
-#            'end_date': forms.SelectDateWidget,
-            'phone_daytime': forms.PhoneNumberInput,
-            'phone_evening': forms.PhoneNumberInput
+            'city': forms.TextInput(attrs={'class': 'medium-grid'}),
```

Comment:
These classes don't appear to be from the USWDS. If you haven't already done so, consider incorporating the "Address form" example from https://standards.usa.gov/form-templates/.